lib/msal-browser/test/naa/JSRuntime.spec.ts

```diff
@@ -39,6 +42,7 @@ describe("JS Runtime Nested App Auth", () => {
         // Remove global properties that can be reset after test that don't have full implementations in JS Runtime
         deleteGlobalProperty("crypto");
         deleteGlobalProperty("TextEncoder");
+        deleteGlobalProperty("BroadcastChannel");
```

Comment:
To catch future issues related to globals I'd recommend creating your own [jest environment](https://jestjs.io/docs/configuration#testenvironment-string) that more closely approximates the JS Runtime